bucket/prismlauncher-qt5.json

```diff
@@ -1,10 +1,21 @@
 {
-    "version": "5.2",
-    "description": "An Open Source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability. Qt 5 build that still supports legacy Windows like 8.1 and 7.",
+    "version": "6.0",
+    "description": "An open sourced Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability. Qt 5 build that still supports legacy Windows like 8.1 and 7.",
```

Comment:
any reason for this change?

Reply:
doesn't have to be capitalised, it's a grammar mistake